### DISCUSSION TITLE
Windows, Mac, Linux, Raspberry Pi ?

### DISCUSSION THREAD

**Discussion by White-Rabbit-Scientific** [israel-dryer/ttkbootstrap#367] [General]
Please can it be made clear whether TtkBootstrap works on Windows, Mac, Linux, Raspberry Pi.

This is a fantastic project imo. Very pleasing to find it today 😁😁😁

**Comment by rdbende** [comment]
Yes, it's platform independent. It works on Windows, macOS and any Linux distro.

**Reply by White-Rabbit-Scientific** [reply]
Nice. It would be great to have "cross platform" made a bit clearer on the front page somewhere. 

(I have now tried it on the Mac and it looks pretty good except for section corners where two thin lines meet at right angles. The junction lacks a nice rounded corner, but a minor niggle.)


<img width="328" alt="image" src="https://user-images.githubusercontent.com/6218191/199553752-c8809b73-2de6-49f8-97bf-5b63848d1b34.png">
